### ISSUE TITLE
how do i change the hue,saturation and all ?

### ISSUE THREAD

**Issue by saikrishnadas** [WongKinYiu/ScaledYOLOv4#95]

**Comment by WongKinYiu** [comment]
https://github.com/WongKinYiu/ScaledYOLOv4/blob/yolov4-csp/data/hyp.scratch.yaml#L17-L19

**Comment by saikrishnadas** [comment]
The main advantage of Scaled-Yolo was its speed. I tried training a p7 model with 2 classes with 500 images each (total 1000 images) , 64 batch-size and 1000 epochs, but i took me around 20hrs to complete the training process which is slower than darknet yolov4 model. Why is it so ? 
I use single Nvidia tesla GPU.

**Comment by WongKinYiu** [comment]
it because p7 model is 5 times larger than yolov4 model.
for similar model scale, please use yolov4-csp.

**Comment by saikrishnadas** [comment]
Okay,
And why do i get multiple predictions on a single object? 
I trained plum and apple as a model and while inference I get multiple predictions on a single object inside the image.

Attaching the images,
![test1](https://user-images.githubusercontent.com/30836867/102308562-e717cc80-3f8c-11eb-8ef5-c7d4b779e45f.jpg)

the one in the top gives 2 predictions

2. Which script and weight i shld use ? Shoud i use detect.py or test.py script ? shld be last_yolov4-p7_strip.pt or last_yolov4-p7.pt (not using best.pt because i may be due to overfiitting )

**Comment by WongKinYiu** [comment]
1. binary cross entropy loss enable multiple label for single bbox, if your case only have one label for one bbox, you can use cross entropy loss.
2. last usually more over-fitting than best.

**Comment by saikrishnadas** [comment]
So, you recommend me to use the best and not last? But sometimes best comes with at 500 epochs from 2000 epochs. !!
And how do I calculate MAP for each weight after the training is completed?

1. Which one is suitable for final production? And what's the difference between a normal and strip weight? 
![Screenshot (135)_LI](https://user-images.githubusercontent.com/30836867/102445552-be0d3f80-4051-11eb-881c-79ffce31facd.jpg)


2. Where do I change the loss function?  I wanted to use Cross entropy instead of binary CE , where do i change it ?
